crates/sui-sdk/src/lib.rs

```diff
@@ -197,25 +394,30 @@ impl Debug for RpcClient {
     }
 }
 
+/// ServerInfo contains all the useful information regarding the version and the available RPC calls and subscriptions
 struct ServerInfo {
     rpc_methods: Vec<String>,
     subscriptions: Vec<String>,
     version: String,
 }
 
 impl SuiClient {
+    /// Returns a list of available RPC calls
     pub fn available_rpc_methods(&self) -> &Vec<String> {
         &self.api.info.rpc_methods
     }
 
+    /// Returns a list of available subscriptions
     pub fn available_subscriptions(&self) -> &Vec<String> {
         &self.api.info.subscriptions
     }
 
+    /// Returns the API version information
```

Comment:
These doc comments seem a bit redundant on their own.  Here's some information that might be more helpful here:

- What is an RPC Method vs a Subscription
- Why do these values change? (i.e. why would a given RPC method be available in some cases but not others).
- What information is included in "API version information" other than the version itself?